gatling-core/src/main/scala/io/gatling/core/assertion/AssertionBuilders.scala

```diff
@@ -66,7 +66,13 @@ class AssertionWithPathAndTarget[T: Numeric](path: AssertionPath, target: Target
   @SuppressWarnings(Array("org.wartremover.warts.DefaultArguments"))
   def around(mean: T, plusOrMinus: T, inclusive: Boolean = true): Assertion =
     between(numeric.minus(mean, plusOrMinus), numeric.plus(mean, plusOrMinus), inclusive)
+  @SuppressWarnings(Array("org.wartremover.warts.DefaultArguments"))
+  def deviatesAround(target: T, percentDeviationThreshold: T, inclusive: Boolean = true): Assertion = {
```

Comment:
IMHO, percentDeviationThreshold should be a Double as it's a percentage.

Reply:
your right, it will be better